### DISCUSSION TITLE
Renommer les topics d'un graphique issu du LDA-segment dans Lodex

### DISCUSSION THREAD

**Discussion by camilledesalabert** [Inist-CNRS/lodex#2082] [Ideas]
Les résultats du WS LDA peuvent être visualisés dans un graphique de type "Diagramme à barres groupées". Malheureusement le paramétrage de ce format est très réduit. 

Il serait intéressant de pouvoir renommer les titres des différents diagrammes de la série par un nom plus explicite que "topic_1", "topic_2", etc.

**Comment by touv** [comment]
le problème est plus général, par exemple en proposant des enrichissements sur le résultat des pré-calculs ou de pouvoir éditer les routines
